parkings/anonymization/__init__.py

```diff
@@ -0,0 +1,55 @@
+from django.conf import settings
+from django.db import transaction
+from django.utils import timezone
+
+from ..models import Parking, ParkingCheck, PermitLookupItem
+
+
+def anonymize_parking_registration_numbers():
+    parkings = (
+        Parking.objects.filter(time_end__lt=get_past_time())
+        .exclude(registration_number="")
+        .order_by("time_end")
+    )
+
+    for parking in parkings:
+        with transaction.atomic():
+            parking.registration_number = ""
+            parking.normalized_reg_num = ""
+            parking.save(update_fields=["registration_number", "normalized_reg_num"])
+
+
+def anonymize_parking_check_registration_numbers():
+    parking_checks = (
+        ParkingCheck.objects.filter(created_at__lt=get_past_time())
+        .exclude(registration_number="")
+        .order_by("created_at")
+    )
+
+    for parking_check in parking_checks:
+        with transaction.atomic():
+            parking_check.registration_number = ""
+            parking_check.save(update_fields=["registration_number"])
+
+
+def anonymize_permit_registration_numbers():
+    parking_lookups = (
```

Comment:
Nitpick ofc, but I guess this variable should be named `permit_lookups`.

Reply:
Will take into account.

Reply:
Taken into account in PR #223 by removing the variable name completely (See https://github.com/City-of-Helsinki/parkkihubi/pull/223/commits/31be2a131ce5a7b815073d6350c626ec0ed54b52#diff-125eddeefa8c7da8082d77756549474a074b240ea65e28f2b6c190f568d64dafL32-R16).